src/dasmxx.c

```diff
@@ -942,6 +946,45 @@ static void run_disasm( struct params params )
             bpl   = clist->bpl;
             clist = clist->n;
         }
+        else if ( mode == WSTRING )
+        {
+            /*****************************************************************
+            *            u - WIDECHAR STRING DATA
+            *****************************************************************/
+
+            int c;
+
+            newline();
+            printcomment( blockcmt, addr, 0 );
+
+            while ( addr < clist->addr )
+            {
+                emitaddr( addr, &params );
+                if ( params.want_asm_out )
+                    printf( params.want_stripped ? "   " : "\n   " );
+                printf( "DW      '" );
+
+                while ( c = nextw( f, &addr ) )
+                {
+                    if ( c == string_terminator )
+                        break;
+
+                    if ( isprint( c ) )
+                        putchar( c );
+                    else
+                        printf ("', %#04X, '", (uint16_t) c );
```

Comment:
I wonder if a better solution to the placement of single quotes would make the layout tidier?  Maybe something like
`int in_quote = 0;`  just before the while loop, remove the first single quote from the DW line, and then the following:
 ```
.
.
.
if ( isprint( c ) )
{
     if ( !in_quote )
     {
          putchar( "'" );
          in_quote = 1;
     }
     putchar( c );
}
else
{
     if ( in_quote )
     {
          puts( "', " );
          in_quote = 0;
     }
     else
     { 
          puts ( ", " );
     }
     printf ( "%#04X", (uint16_t) c );
}
.
.
.
```
And then finally
```
if ( in_quote )
    printf( "'" );
```